minio/minio
# Make sure to trace generic-handlers as well

## Description
Make sure to trace generic-handlers as well

## Motivation and Context
Allow tracing of generic-handler calls

## How to test this PR?
Send a malformed request to MinIO with an incorrect Authorization header

```
~ curl -H "Authorization: Unknown" http://localhost:9000
```

You shall see "InvalidRequest" but with the current master, you cannot trace these operations. This PR fixes this behavior.


## Types of changes

## Checklist:
